webpack.config.js

```diff
@@ -176,43 +188,26 @@ module.exports = (env, argv) => ({
     ],
     noParse: [/node_modules\/sinon\//],
   },
-  externals: {
-    'pixi.js': {
-      commonjs: 'pixi.js',
-      commonjs2: 'pixi.js',
-      amd: 'pixi.js',
-      root: 'PIXI',
-    },
-    react: {
-      commonjs: 'react',
-      commonjs2: 'react',
-      amd: 'react',
-      root: 'React',
-    },
-    'react-dom': {
-      commonjs: 'react-dom',
-      commonjs2: 'react-dom',
-      amd: 'react-dom',
-      root: 'ReactDOM',
-    },
-  },
```

Comment:
I'd recommend keeping these dependencies external – although I know it seems to cause headaches. Ideally, `higlass` should bundle it's dependencies, but since the export is a UMD module it seems necessary.

In Gosling, we use a different version of react/react-dom and PIXI (but we rely on accessing `PIXI` from `HGC.libraries.PIXI`). Changing this behavior means we would no longer specify the version of PIXI we are using. 

Additionally (and the bigger concern) is that we extend the `HiGlassComponent` directly in `gosling.js`, and render our own components with `react`/`react-dom`. From [the docs](https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react),

> In order for Hooks to work, the `react` import from your application code needs to resolve to the same module as the `react` import from inside the `react-dom` package.

Not externalizing `react` & `react-dom` leads to multiple copies React that we can't resolve since the code is bundled.